cmake/Modules/FindCatch_EP.cmake

```diff
@@ -75,6 +91,8 @@ endif()
 if (CATCH2_FOUND AND NOT TARGET Catch2::Catch2)
   add_library(Catch2::Catch2 INTERFACE IMPORTED)
   set_target_properties(Catch2::Catch2 PROPERTIES
```

Comment:
This is archaic CMake syntax. `target_include_directories` would work better. It might also solve the blocking problem of all the should-be-redundant calls to this same function for unit test targets.

Reply:
block removed along with addition of searching for or building specific minimum version that should always have needed targets.